### PR TITLE
chore: replace the files action

### PR DESCRIPTION
**Pull request by dhavaldodiya**

### CODE REVIEW

**Review by gnekoz — DISMISSED** [review]

File: src/view/editor/parts/editor-attachments-button.tsx [hunk 1]
```diff
@@ -100,24 +97,22 @@ export const EditorAttachmentsButton = ({ editorId, callbacks }: EditorProps): R
...
-		'carbonio_files_action',
-		'files-select-nodes',
-		actionTarget
-	);
+	const [filesSelectFilesAction, filesSelectFilesActionAvailable] =
+		getIntegratedFunction('select-nodes');
 
 	const addFileItem = useMemo(
 		() => ({
 			...filesSelectFilesAction,
```
Comment (CataldoMazzilli) [inline src/view/editor/parts/editor-attachments-button.tsx]: this destructuring is no more required

**Review by gnekoz — DISMISSED** [review]

File: src/view/editor/parts/editor-attachments-button.tsx [hunk 2]
```diff
@@ -100,24 +97,19 @@ export const EditorAttachmentsButton = ({ editorId, callbacks }: EditorProps): R
...
-	const [filesSelectFilesAction, filesSelectFilesActionAvailable] = getAction(
-		'carbonio_files_action',
-		'files-select-nodes',
-		actionTarget
-	);
-
 	const addFileItem = useMemo(
 		() => ({
-			...filesSelectFilesAction,
+			...getFilesAction,
```
Comment (CataldoMazzilli) [inline src/view/editor/parts/editor-attachments-button.tsx]: remove "...getFilesAction" it is a function

File: src/view/editor/parts/editor-attachments-button.tsx [hunk 3]
```diff
@@ -70,23 +70,20 @@ export const EditorAttachmentsButton = ({ editorId, callbacks }: EditorProps): R
...
 
-	const [getFilesAction, getFilesActionAvailable] = getAction(
-		'carbonio_files_action',
-		'files-select-nodes',
-		actionURLTarget
-	);
+	const [getFilesAction, getFilesActionAvailable] = getIntegratedFunction('select-nodes');
 	const publicUrlItem = useMemo(
 		() => ({
 			...getFilesAction,
```
Comment (CataldoMazzilli) [inline src/view/editor/parts/editor-attachments-button.tsx]: remove "...getFilesAction" it is a function

**Review by CataldoMazzilli — CHANGES_REQUESTED** [review]
build is failing

**Review by nubsthead — DISMISSED** [review]

**Review by CataldoMazzilli — DISMISSED** [review]

**Review by gnekoz — DISMISSED** [review]

**Review by giuliano176 — APPROVED** [review]

**Review by CataldoMazzilli — APPROVED** [review]

**Review by nubsthead — APPROVED** [review]

**Pull request merged**